portfolio/src/main/webapp/script.js

```diff
@@ -54,7 +54,12 @@ function getJSONMessages() {
   fetch('/data').then(response => response.json()).then((messages) => {
     const msgElement = document.getElementById('message-container');
     msgElement.innerHTML = '';
-    msgElement.innerText = messages[0] + '\n' + messages[1] + '\n' + messages[2];
+    console.log(messages);
```

Comment:
This is fine for testing functionality, but log statements like this should be omitted from the final code.

Reply:
whoops did not mean to keep that in there!